pyvista/core/utilities/geometric_sources.py

```diff
@@ -2249,6 +2269,98 @@ def j_resolution(self, j_resolution: int):
         """
         self.SetYResolution(j_resolution)
 
+    @property
+    def center(self) -> Sequence[float]:
+        """Get the center in ``[x, y, z]``.
+
+        Returns
+        -------
+        sequence[float]
+            Center in ``[x, y, z]``.
+        """
+        return self.GetCenter()
+
+    @center.setter
+    def center(self, center: Sequence[float]):
+        """Set the center in ``[x, y, z]``.
+
+        Parameters
+        ----------
+        center : sequence[float]
+            Center in ``[x, y, z]``.
+        """
+        self.SetCenter(center)
+
+    @property
+    def origin(self) -> Sequence[float]:
+        """Get the origin in ``[x, y, z]``.
+
+        Returns
+        -------
+        sequence[float]
+            Origin in ``[x, y, z]``.
+        """
+        return self.GetOrigin()
+
+    @origin.setter
+    def origin(self, origin: Sequence[float]):
+        """Set the origin in ``[x, y, z]``.
+
+        Parameters
+        ----------
+        origin : sequence[float]
+            Origin in ``[x, y, z]``.
+        """
+        self.SetOrigin(origin)
+
+    @property
+    def pointa(self) -> Sequence[float]:
+        """Get the Location in ``[x, y, z]``.
+
+        Returns
+        -------
+        sequence[float]
+            Location in ``[x, y, z]``.
+        """
+        return self.GetPoint1()
+
+    @pointa.setter
+    def pointa(self, pointa: Sequence[float]):
+        """Set the Location in ``[x, y, z]``.
+
+        Parameters
+        ----------
+        pointa : sequence[float]
+            Location in ``[x, y, z]``.
+        """
+        if np.array(pointa).size != 3:
+            raise TypeError('Point A must be a length three tuple of floats.')
```

Comment:
The other params don't have this check, vtk will handle this otherwise.
```suggestion
```

Though  if wanting to validate I otherwise suggest 
```python
_validation.validate_array3(point_a, dtype_out=float, name='point_a')
```
and using `VectorLike[float]` typing

Reply:
Let's delete it and add if we need them later.